### DISCUSSION TITLE
CrustyAuklet

### DISCUSSION THREAD

**Discussion by cschreib** [snitch-org/snitch#162] [On-boarding]
On-boarding checklist for @CrustyAuklet
 - [x] User satisfies the criteria for inclusion, as described in the [contribution documentation](https://github.com/snitch-org/snitch/blob/main/CONTRIBUTING.md), and has not been previously banned from this organization (you can search for their username in this discussion category, to see if they have been previously on-boarded; make sure to search in closed discussions).
 - [x] User has been invited to the organization. To do so, head to the [People](https://github.com/orgs/snitch-org/people) tab of the organization, then select "Invite member". Type in their GitHub username, then select "Invite".
 - [x] User has accepted the invitation.
 - [x] User has responded to this discussion, confirming they have read the [code of conduct](https://github.com/snitch-org/snitch/blob/main/CODE_OF_CONDUCT.md) and the [contribution documentation](https://github.com/snitch-org/snitch/blob/main/CONTRIBUTING.md). 
 - [x] User is added to the maintainer team.
 - [x] (optional) User has [made their membership public](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership). This is not mandatory, but helps other users gain confidence in the project if they can see all maintainers.

**Comment by CrustyAuklet** [comment]
I went over the code of conduct and contribution documentation, all sounds good. I do appreciate the contribution and review process I have experienced so far. The current snitch-org team was really patient at helpful!

**Reply by cschreib** [reply]
Thanks for the feedback! And welcome ;)
